// pubg-stats/src/MatchHistory.js
class PUBGProfileMatchHistory {
	constructor(content) {
		this.id = content.Id;
		this.updated = content.Updated;
		this.season = content.SeasonDisplay;
		this.seasonRaw = content.Season;
		this.match = content.MatchDisplay;
		this.matchRaw = content.Match;
		this.region = content.RegionDisplay;
		this.regionRaw = content.Region;
		this.rounds = content.Rounds;
		this.wins = content.Wins;
		this.kills = content.Kills;
		this.assits = content.Assists;
		this.top10 = content.Top10;
		this.rating = content.Rating;
		this.ratingChange = content.RatingChange;
		this.ratingRank = content.RatingRank;
		this.ratingRankChange = content.RatingRankChange;
		this.headshots = content.Headshots;
		this.kd = content.Kd;
		this.damage = content.Damage;
		this.timeSurvived = content.TimeSurvived;
		this.winRating = content.WinRating;
		this.winRank = content.WinRank;
		this.winRatingChange = content.WinRatingChange;
		this.winRatingRankChange = content.WinRatingRankChange;
		this.killRating = content.KillRating;
		this.killRank = content.KillRank;
		this.killRatingChange = content.KillRatingChange;
		this.killRatingRankChange = content.KillRatingRankChange;
		this.moveDistance = content.MoveDistance;
	}
}

module.exports = PUBGProfileMatchHistory;
